cdap-examples/UserProfiles/README.md

```diff
@@ -0,0 +1,72 @@
+# User Profiles Example
+
+Example application that manages user profiles in a Table. The fields of a user
+profile are updated in different ways:
+
+- Attributes like name and email address are changed through a REST
+  call when the user updates his profile;
+- The time of the last login is updated by a sign-on service every
+  time the user logs in, also through a REST call;
```

Comment:
The time of the last login is updated by a sign-on service every
time the user logs in, also through a RESTful call; and